universal_transfer_operator/example_dags/example_universal_transfer_operator.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+
+from airflow import DAG
+from datasets.file import File
+from datasets.table import Table
+from universal_transfer_operator import UniversalTransferOperator
+
+from constants import TransferMode
+
+START_DATE = datetime(2000, 1, 1)
+with DAG(
+    "example_universal_transfer_operator",
+    schedule_interval=None,
+    start_date=START_DATE,
+    catchup=False,
+) as dag:
+    transfer_non_native = UniversalTransferOperator(
+        task_id="transfer_non_native",
+        source_dataset=File("gs://uto-test/uto/", conn_id="google_cloud_default", extra={}),
+        destination_dataset=File("s3://astro-sdk-test/uto/", conn_id="aws_default", extra={}),
+    )
+
+    transfer_fivetran_with_connector_id = UniversalTransferOperator(
+        task_id="transfer_fivetran_with_connector_id",
+        source_dataset=File("s3://astro-sdk-test/uto/", conn_id="aws_default", extra={}),
+        destination_dataset=Table(
+            "snowflake://gp21411.us-east-1.snowflakecomputing.com/providers_fivetran_dev.s3.fivetran_ankit_test",
```

Comment:
There is a duplication of this info from connection -- schema and other things, no?

Reply:
Yes, that's correct but we need a pointer to the table and Open lineage has a really good way of defining that: https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md

Reply:
We can generate that internally but let's not duplicate those details

Reply:
Made the change: https://github.com/astronomer/astro-sdk/pull/1492/commits/b59f087700a88b7f24989d0619c89bd06771c8e5

Removed `path` and `conn_id` from dataset. Added `name` to `Table`